base_technical_user/tests/test_sudo_tech.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests import SavepointCase
+from odoo.exceptions import UserError
+from odoo import SUPERUSER_ID
+
+
+class SudoTechCase(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(SudoTechCase, cls).setUpClass()
+        cls.user_tech = cls.env["res.users"].create({"login": "tech", "name": "tech"})
```

Comment:
always use `tracking_disable` + `no_reset_password` when creating users in tests ;)

Reply:
done